modules/messages/src/lib.rs

```diff
@@ -37,56 +37,66 @@
 // Generated by `decl_event!`
 #![allow(clippy::unused_unit)]
 
+#[cfg(test)]
+mod mock;
+
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmarking;
+
+pub mod weights;
 pub use weights::WeightInfo;
+
+pub mod instant_payments;
```

Comment:
We can delete `instant_payments` mod, this is replaced by fee market payment in our system.

Reply:
Good catch.